model/config.go

```diff
@@ -962,23 +962,23 @@ type PasswordSettings struct {
 
 func (s *PasswordSettings) SetDefaults() {
 	if s.MinimumLength == nil {
-		s.MinimumLength = NewInt(PASSWORD_MINIMUM_LENGTH)
+		s.MinimumLength = NewInt(10)
```

Comment:
Why can't you use the constant here?

Reply:
The difference is that `PASSWORD_MINIMUM_LENGTH` is really the lower bound what is allowed to be set in the config and in passwords, if we would use the constant here and adapt it, it would effectively mean that Mattermost does not allow any password policy below 10 characters.

e.g.:
https://github.com/mattermost/mattermost-server/blob/0c807432a933b746eb064bc1a4d76f4f03e55bf3/model/config.go#L2432

But we want to set the default to 10, and allow new customers and also existing customers that have it already set this way to use also 5-9 characters if this is what they desire (essentially weakening the security obviously).

Reply:
Ah I see. Fine with me then.